modules/ROOT/pages/ip-whitelist.adoc

```diff
@@ -7,68 +7,66 @@ endif::[]
 [width="100%", cols="5,15"]
 |===
 >s| Policy Name | IP Whitelist
->s|Summary      | Allows request only from a list or range of IP addresses
+>s|Summary      | Allows access to request from a list or range of specified IP addresses
```

Comment:
"Allows a list or range of specified IP addresses to request access"
OR
"Allows a list or range of specified IP addresses to request access to" <this>

Reply:
Done! Thanks!